scripts/rv_expression_association/README.md

```diff
@@ -0,0 +1,98 @@
+## Analysis plan
+
+The overall goal of these scripts is to run a gene-set association test to test for an association between i) a number of rare genetic variants located in and around a gene and ii) the expression level of the gene itself (in a given cell type).
```

Comment:
This runs a gene-set association analysis to test for an association between i) the number of rare genetic variants located in and 50kb up or down-stream of the gene and ii) the expression level of the gene itself (in a given cell type).